chemdyg/templates/chemdyg_STE_flux_diags.bash

```diff
@@ -71,30 +69,9 @@ do
   done
 done
 
-#{%- if frequency != 'monthly' %}
-## For non-monthly input files, need to add the last file of the previous year
-#year={{ year1 - 1 }}
-#YYYY=`printf "%04d" ${year}`
-#mapfile -t files < <( ls ${input}/{{ case }}.{{ input_files }}.${YYYY}-*.nc 2> /dev/null )
-#{% raw -%}
-#if [ ${#files[@]} -ne 0 ]
-#then
-#  ln -s ${files[-1]} .
-#fi
-#{%- endraw %}
-# as well as first file of next year to ensure that first and last years are complete
-#year={{ year2 + 1 }}
-#YYYY=`printf "%04d" ${year}`
-#mapfile -t files < <( ls ${input}/{{ case }}.{{ input_files }}.${YYYY}-*.nc 2> /dev/null )
-#{% raw -%}
-#if [ ${#files[@]} -ne 0 ]
-#then
-#  ln -s ${files[0]} .
-#fi
-#{%- endraw %}
-#{%- endif %}
-
-#cd ..
+mapDir="{{ diagnostics_base_path }}/observations/Atm/ChemDyg_inputs"
+ln -s ${mapDir}/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc ./
```

Comment:
This map file only works for ne30pg2 files. Should it be a cfg input, so users can specify the mapping files by themselves?